apps/docs/pages/core/modules/custom.mdx

```diff
@@ -0,0 +1,73 @@
+import { Callout, Steps } from "nextra/components";
+
+# Using the Custom Module in Wallet API Client
+
+The Custom Module allows you to interact with any custom handler supported by the server.
+
+## Custom Module Overview
+
+<Callout type="info" emoji="ℹ️">
+  We are still working on improving the type safety of custom modules
+</Callout>
+
+<Steps>
+### Step 1
+
+Create the WalletAPIClient with the `getCustomModule` param.
```

Comment:
I guess we should specify that we're referring here to the (https://wallet.api.live.ledger.com/core/configuration) phase.

And add something along the lines of:

"First, go back to your transport declaration from your Wallet API Configuration.
You need to adapt the WalletAPIClient by passing a callback that returns your custom handler"

Reply:
Will do